mks/model/stuf_02_04.py

```diff
@@ -314,7 +320,38 @@ def extract_verbintenis_data(persoon_tree: Tag):
 
 
 def extract_address(persoon_tree: Tag, is_amsterdammer):
-    result = []
+    addresses_vbl_source = persoon_tree.find_all("PRSADRVBL")
+    addresses_cor_source = persoon_tree.find_all("PRSADRCOR")
+    addresses_vbl = format_addresses(
+        addresses_vbl_source, ADDR_VBL, is_amsterdammer)
+    addresses_cor = format_addresses(
+        addresses_cor_source, ADDR_COR, is_amsterdammer)
+    addresses = merge_sort_addresses(addresses_vbl, addresses_cor)
+
+    return addresses
+
+
+def merge_sort_addresses(*address_lists):
+    all_addresses = list(chain(*address_lists))
+    if is_nil(all_addresses):
+        return {}, []
+    all_addresses.sort(
+        key=lambda x: x["begindatumVerblijf"] or date.min, reverse=True)
+    end = all_addresses[0]["einddatumVerblijf"]
+    hasCurrent = end is None or end > date.today()
+    # Laatste inschrijfdatum
+    address_current = all_addresses[0] if hasCurrent else {}
+    # Alle andere, beginnend met het vorige adres
+    address_history = all_addresses[1:] if hasCurrent else all_addresses[0:]
+    for address in address_history:
+        if address.get("_adresSleutel"):
+            del address["_adresSleutel"]
```

Comment:
Dit codeblock kun je denk ik beter in een andere functie zetten of plaatsen na mergen en sorten in `extract_address(..)`